go/cmd/dolt/commands/merge.go

```diff
@@ -252,6 +263,10 @@ func executeMerge(ctx context.Context, dEnv *env.DoltEnv, cm1, cm2 *doltdb.Commi
 
 		if hasConflicts {
 			cli.Println("Automatic merge failed; fix conflicts and then commit the result.")
+		} else {
+			// Fail silently because merge was still successful.
+			// If staged root failed to update, the tables can still be added and committed to conclude merge.
+			UpdateStagedWithVErr(dEnv, mergedRoot)
```

Comment:
Never want to fail silently if you can help it. If it's a non-fatal error, log a warning message.

Reply:
Removed silent fail, updated it to this: https://github.com/liquidata-inc/dolt/pull/299/files#diff-1647ea59f1bf8a37af76a621b0ea152fR267